system.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+// StopEtcd stops etcd on a specific host
+func StopEtcd(node TestbedNode) error {
+	log.Infof("Stopping etcd")
+	return node.RunCommand("pkill etcd && rm -rf /tmp/etcd")
+}
+
+// StartEtcd starts etcd on a specific host.
+func StartEtcd(node TestbedNode) error {
+	log.Infof("Starting etcd")
+
+	_, err := node.RunCommandBackground("nohup etcd -data-dir /tmp/etcd </dev/null &>>/tmp/etcd.log &")
```

Comment:
@erikh,  I think we should truncate the logs instead of appending them, otherwise it will be hard to separate out the logs for a particular run when a testcase fails. WDYT?

We can go a bit fancy and add a pid or timestamp to the log filename to keep logs from across different starts but I am not sure if it is ver useful.